src/pages/library/datasetBuilder/CohortEditor.test.ts

```diff
@@ -178,8 +178,12 @@ describe('CohortEditor', () => {
       updateCriteria,
     });
     // Act
-    screen.getByLabelText(`${criteria.name} low slider`);
-    screen.getByLabelText(`${criteria.name} high slider`);
+    fireEvent.keyDown(screen.getByLabelText(`${criteria.name} low slider`), { keyCode: 39 /* Right Arrow */ });
```

Comment:
Is there a JS builtin or library that provides definitions for key codes? It'd be nice to avoid magic numbers if possible.